src/server/menu.js

```diff
@@ -41,6 +41,7 @@ global.onOpen = () => {
   submenuDeveloper.addItem('Show spreadsheet developer metadata', 'showSpreadsheetAllDevMetadata');
   submenuDeveloper.addItem("Set current sheet's last used schema version", 'setLastUsedSchemaVersion');
   submenuDeveloper.addItem('Authorize for ENCODE', 'authorizeForEncode');
+  // submenuDeveloper.addItem('Test Google Drive', 'testGoogleDrive');
```

Comment:
should not this comment be removed?

Reply:
yeah, I will remove it but I have other redundant commented lines in the code.
I will remove them all in the new ticket `WRN-459`.